earth_enterprise/src/fusion/portableglobe/servers/fileunpacker/Windows/khTypes.h

```diff
@@ -22,11 +22,8 @@
 #include <assert.h>
 #include <sys/types.h>
 
-typedef unsigned short                  uint16;
 typedef short                           int16;
```

Comment:
should you remove these too?  It would be more portable if the code didn't use these anymore and used `std` versions instead.